HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -3077,6 +3080,50 @@ def self.set_cool_rated_shrs_gross(cooling_system)
     end
   end
 
+  def self.apply_supp_control_for_ddb_thermostat(model, supp_coil, control_zone)
```

Comment:
@jmaguire1 Please take a look at this. I added an EMS control to limit supplemental heating coil operation, so that it is only enabled when zone temperature is out of deadbands.